src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from "react";
+
+import {
+  isCurrentVenueRequestedSelector,
+  isCurrentVenueRequestingSelector,
+  venueSelector,
+} from "utils/selectors";
+
+import { IFRAME_TEMPLATES } from "settings";
+
+import { useSelector } from "hooks/useSelector";
+import { useUserIsVenueOwner } from "hooks/useUserIsVenueOwner";
+import useConnectCurrentVenue from "hooks/useConnectCurrentVenue";
+import { useUser } from "hooks/useUser";
+
+import { LoadingPage } from "components/molecules/LoadingPage/LoadingPage";
+import { AdminVideo } from "components/molecules/AdminVideo";
+import { AdminBanner } from "components/molecules/AdminBanner/AdminBanner";
+
+import "./VenueAdminPage.scss";
+
+export const VenueAdminPage: FC = () => {
+  useConnectCurrentVenue();
+
+  const { profile, user } = useUser();
+  const venue = useSelector(venueSelector);
+  const venueRequestStatus = useSelector(isCurrentVenueRequestedSelector);
+  const venueRequestingStatus = useSelector(isCurrentVenueRequestingSelector);
+
+  const isVenueOwner = useUserIsVenueOwner();
+  const isLoadingVenue = venueRequestingStatus || !venueRequestStatus;
+  const isLoggedIn = profile && user;
+
+  if (isLoadingVenue) {
+    return <LoadingPage />;
+  }
+
+  if (!venue) {
+    return <div className="admin-page-title">This venue does not exist</div>;
+  }
+
+  if (!isLoggedIn) {
+    return <div className="admin-page-title">You need to log in first.</div>;
+  }
+
+  if (!isVenueOwner) {
+    return (
+      <div className="admin-page-title">{`You don't have the permissions to access this page`}</div>
+    );
+  }
+
+  const isVideoVenue = IFRAME_TEMPLATES.includes(venue.template);
+
+  return (
+    <>
```

Comment:
Do we really want these elements to be output as is (which is what this fragment will do)? Or do we want them to be wrapped in a 'containing `div`'?

Reply:
Since there are no styles there is no point in rendering an extra element in the DOM. If someone needs to style these he can just replace the fragment with `div`

Reply:
Makes sense